tests/installation/partitioning_raid.pm

```diff
@@ -152,43 +149,60 @@ sub set_lvm {
     type_string "root";
     assert_screen 'volume-name-root';
     send_key $cmd{next};
-
-    # keep default
     send_key $cmd{next};
 
+    assert_screen 'volume-pick-fs-role';
     send_key "alt-o";    # Operating System
     send_key $cmd{next};
+    if (is_storage_ng) {
+        if (check_screen 'volume-empty-mount') {
+            # storage-ng does not suggest mount point for OS role volume,
+            # so mount the volume as root explicitely.
+            send_key "alt-o";
+            assert_screen 'volume-mount-as-root';
+        }
+        send_key $cmd{next};
+    }
 
     # keep default to mount as root and btrfs
-    wait_screen_change { send_key $cmd{finish} };
+    wait_screen_change { send_key is_storage_ng() ? $cmd{next} : $cmd{finish}; };
 }
 
 sub modify_uefi_boot_partition {
+    send_key 'tab' if is_storage_ng;
     assert_screen 'partitioning_raid-disk_vdd_with_partitions-selected';
     # fold the drive tree
     send_key 'left';
     assert_screen 'partitioning_raid-hard_disks-unfolded';
     # select first disk
+    if (is_storage_ng) {
+        send_key 'left';
+        send_key 'right';
+    }
     send_key 'right';
     assert_screen 'partitioning_raid-disk_vda_with_partitions-selected';
     # edit first partition
     send_key 'alt-e';
     assert_screen 'partition-format';
-    # format as FAT (first choice)
     send_key 'alt-a';
-    assert_screen 'partitioning_raid-format_fat_UEFI';
+    assert_screen is_storage_ng() ? 'partitioning_raid-format_ext_UEFI' : 'partitioning_raid-format_fat_UEFI';
```

Comment:
Looks like we have different default values, WDYT if we create needle called partitioning_raid-format_default_UEFI here?

Reply:
Sure: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/641.